Sources/ProjectDescription/Cache.swift

```diff
@@ -41,7 +41,8 @@ public struct Cache: Codable, Equatable {
 
     /// Returns a `Cache` instance containing the given profiles.
     /// If no profile list is provided, tuist's default profile will be taken as the default.
-    /// If no profile is provided in `tuist cache --profile` command, the first profile from the profiles list will be taken as the default.
+    /// If no profile is provided in `tuist cache --profile` command, the first profile from the profiles list will be taken as
+    /// the default.
```

Comment:
@Dmitry-Pliushchai can you clean up the PR from mall the unrelated changes like this or the gems? 🙏